pkg/plugin/prommetrics.go

```diff
@@ -124,3 +132,7 @@ func (m *PromMetrics) IncFailIfNotSuccess(method string, pod *corev1.Pod, ignore
 
 	m.pluginCallFails.WithLabelValues(method, util.PodPreferredAZIfPresent(pod), strconv.FormatBool(ignored), status.Code().String())
 }
+
+func (m *PromMetrics) ReserveShouldDeny(pod *corev1.Pod, node *nodeState) {
```

Comment:
```suggestion
func (m *PromMetrics) IncReserveShouldDeny(pod *corev1.Pod, node *nodeState) {
```

To match other methods, making it more clear that this increments the metric